src/mon/MonCommands.h

```diff
@@ -1164,6 +1164,13 @@ COMMAND("osd force_recovery_stretch_mode " \
 	"pool size to its non-failure value if currently degraded and "
 	"all monitor buckets are up",
 	"osd", "rw")
+COMMAND("osd set-allow-crimson " \
+	"name=yes_i_really_mean_it,type=CephBool,req=false",
+	"Allow crimson-osds to boot and join the cluster.  Note, crimson-osd is "
+	"not yet considered stable and may crash or cause data loss -- should "
+	"be avoided outside of testing and development.  This setting is "
+	"irrevocable",
```

Comment:
`or` instead `xor` in the incremental map handling? Do we have a plan to change this claim in the future?

Reply:
It's irrevocable for now as a matter of policy, but we'll probably relax that restraint in the future.